lib/bambora/rest/batch_payment_file_upload_client.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+module Bambora
+  module Rest
+    class BatchPaymentFileUploadClient < Bambora::Rest::Client
+      CONTENT_DISPOSITION = 'form-data'
+      FILE_CONTENT_TYPE = 'text/plain'
```

Comment:
`FILE_CONTENT_TYPE` and `JSON_CONTENT_TYPE` naming is a bit not in agreement and would map use fit a bit better here?